### ISSUE TITLE
[Request] Mixer Chat or Mix it Up bot integration

### ISSUE THREAD

**Issue by Tohur-Xbl** [rivafarabi/deckboard#260]
Hi was just wondering if it would be possible to add Mix it Up bot [https://github.com/SaviorXTanren/mixer-mixitup](Mix it Up Github) support to Deckboard. Some of us are Mixer Streamers and would love to have some sort of integration with some the tools we use other than the basics. like Mix it up has built in support for Stream Deck to be able to run bot commands  and such from Stream Deck would be awesome to be able to do the same with Deckboard either direct rectly  integrating with Mix it Up via their Dev API built into the bot or either implement Mixer chat support to run stuff that way or both :) at the very least we should get a Mixer integration considering we have a Twitch one.

**Comment by Avengalyne** [comment]
I COMPLETELY agree with this. being able to control Mixitup from Deckboard would be a huge help.
